pyglet/util.py

```diff
@@ -16,25 +18,40 @@ def asbytes(s):
         return bytes(s)
 
 
-def asbytes_filename(s):
-    if isinstance(s, bytes):
+def asbytes_filename(s: Union[str, Buffer]) -> Optional[bytes]:
```

Comment:
This function shouldn't return None, so it was broken.
That said, lets just delete this function entirely. It was only used in one specific place, but I just inlined the code to save on bloat.

Reply:
Rebased, removed, and force-pushed. :+1: